### PR TITLE
suite run dir bin path, and cylc run dir doc tweak.

### PR DESCRIPTION
**Pull request by hjoliver**
Trivial companion to https://github.com/metomi/rose/pull/2252

### CODE REVIEW

File: lib/cylc/job.sh [hunk 1]
```diff
@@ -125,11 +125,8 @@ cylc__job__main() {
     # Send task started message
     cylc message -- "${CYLC_SUITE_NAME}" "${CYLC_TASK_JOB}" 'started' &
     CYLC_TASK_MESSAGE_STARTED_PID=$!
-    # Access to the suite bin directory
-    if [[ -n "${CYLC_SUITE_DEF_PATH:-}" && -d "${CYLC_SUITE_DEF_PATH}/bin" ]]
-    then
-        export PATH="${CYLC_SUITE_DEF_PATH}/bin:${PATH}"
-    fi
+    # Access to the suite bin directory (installed run-dir first).
+    export PATH="${CYLC_SUITE_RUN_DIR}/bin:${CYLC_SUITE_DEF_PATH}/bin:${PATH}"
```
Comment (hjoliver) [inline lib/cylc/job.sh]: `rose suite-run` deploys suite `bin/` for source dir to the run directory.

File: doc/src/cylc-user-guide/siterc.tex [hunk 2]
```diff
@@ -436,22 +436,31 @@ \subsection{[hosts]}
 
 \paragraph[run directory]{[hosts] \textrightarrow [[HOST]] \textrightarrow run directory }
 
-The top level of the directory tree that holds suite-specific output logs,
-run database, etc.
+The top level for suite logs and service files, etc. Can contain
+\lstinline=$HOME= or \lstinline=$USER= but not other environment variables (the
+item cannot actually be evaluated by the shell on HOST before use, but the
+remote home directory is where \lstinline=rsync= and \lstinline=ssh= naturally
+land, and the remote username is known by the suite daemon).
```
[the remark this answers is not in the document]
Comment (hjoliver) [reply doc/src/cylc-user-guide/siterc.tex]: I think we agreed, when I was in Exeter, that changing from "suite daemon" to "suite server program" was probably a mistake (albeit a very small one!) ... it doesn't exactly roll off the tongue. So I've been reverting back, in my own mind.  But OK, we should be consistent at the least, so I'll change this!

File: doc/src/cylc-user-guide/siterc.tex [hunk 3]
```diff
@@ -436,22 +436,31 @@ \subsection{[hosts]}
...
 
 \paragraph[work directory]{[hosts] \textrightarrow [[HOST]] \textrightarrow work directory }
 \label{workdirectory}
 
-The top level for suite work and share directories.
+The top level for suite work and share directories. Can contain
+\lstinline=$HOME= or \lstinline=$USER= but not other environment variables
+(the item cannot actually be evaluated by the shell on HOST before use, but the
+remote home directory is where \lstinline=rsync= and \lstinline=ssh= naturally
+land, and the remote username is known by the suite daemon).
```
[the remark this answers is not in the document]
Comment (hjoliver) [reply doc/src/cylc-user-guide/siterc.tex]: ditto

**Comment by matthewrmshin** [comment]
@kinow please sanity check.

**Comment by kinow** [comment]
Looks good to me. Safe to merge before https://github.com/metomi/rose/pull/2252?

**Comment by matthewrmshin** [comment]
Should be safe.

**Pull request merged**